packages/dashboard/src/components/map-app.tsx

```diff
@@ -316,6 +319,11 @@ export const MapApp = styled(
             return;
           }
           Object.entries(fleetState.robots).forEach(([robotName, robotState]) => {
+            setCurrentLevelOfRobots((prevState) => ({
+              ...prevState,
+              [robotName]: robotState.location?.map || 'L1',
```

Comment:
We should not set a default since there might be maps that do not have `L1` at all, I would suggest just not adding the robot into this record if the location is not available, since that is an issue that needs some attention. 

So in the case where it was already in the record, and somehow its location is missing, we should remove it from this record, and not render it. Probably should log an error too